vault/audit_broker.go

```diff
@@ -110,6 +153,17 @@ func (a *AuditBroker) Deregister(ctx context.Context, name string) error {
 		return fmt.Errorf("%s: name is required: %w", op, event.ErrInvalidParameter)
 	}
 
+	// If the backend isn't actually registered, then there's nothing to do.
+	// We don't return any error so that Deregister can be idempotent.
+	if !a.isRegistered(name) {
+		return nil
+	}
+
+	name = strings.TrimSpace(name)
```

Comment:
This TrimSpace mutation was already done above.

Reply:
Thanks!